### PR TITLE
3466 3467 Add `channel_wise` and correct doc-string

### LINKED ISSUE #3466: `NormalizeIntensity` docstring error
docstring of `channel_wise` is wrong, and it may mislead users that this parameter only works when subtrahend and divisor are not provided.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #3466 #3467 .

### Description
This PR added the `channel_wise` arg to intensity transform and corrected some wrong doc-string.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/transforms/intensity/array.py [hunk 1]
```diff
@@ -967,6 +968,18 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         img = scalar(img)
         return img
 
+    def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
+        """
+        Apply the transform to `img`.
+        """
+        if self.channel_wise:
+            for i, d in enumerate(img):
+                img[i] = self._normalize(img=d)  # type: ignore
```
Comment (wyli) [inline monai/transforms/intensity/array.py:977]: in this case we should probably introduce `axis` to percentile for multichannel computation

**Review by wyli — APPROVED** [review]

File: monai/transforms/intensity/array.py (hunk 1 above)
Comment (Nic-Ma) [reply monai/transforms/intensity/array.py:977]: Sure, added `dim` in the latest commit.

Thanks.

**Pull request merged**